pkg/etcdutil/etcdutil_test.go

```diff
@@ -187,3 +188,39 @@ func (s *testEtcdutilSuite) TestEtcdKVGet(c *C) {
 	c.Assert(len(resp.Kvs), Equals, 2)
 	cleanConfig(cfg)
 }
+
+func (s *testEtcdutilSuite) TestEtcdKVPutWithTTL(c *C) {
+	cfg := newTestSingleConfig()
+	etcd, err := embed.StartEtcd(cfg)
+	c.Assert(err, IsNil)
+
+	ep := cfg.LCUrls[0].String()
+	client, err := clientv3.New(clientv3.Config{
+		Endpoints: []string{ep},
+	})
+	c.Assert(err, IsNil)
+
+	<-etcd.Server.ReadyNotify()
+
+	_, err = EtcdKVPutWithTTL(context.TODO(), client, "test/ttl1", "val1", 5)
+	c.Assert(err, IsNil)
+	_, err = EtcdKVPutWithTTL(context.TODO(), client, "test/ttl2", "val2", 10)
+	c.Assert(err, IsNil)
+
+	time.Sleep(6 * time.Second)
```

Comment:
Can we reduce the sleep time?

Reply:
Yes, and I think these two tests looks similiar, maybe I can factor sth. out.

Reply:
This comment is still not addressed

Reply:
Sorry I didn't notice it, fixed now.